fe/fe-core/src/main/java/com/starrocks/persist/WorkGroupEntry.java

```diff
@@ -0,0 +1,72 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.persist;
+
+import com.starrocks.common.io.Writable;
+import com.starrocks.thrift.TWorkGroup;
+import com.starrocks.thrift.TWorkGroupOp;
+import com.starrocks.thrift.TWorkGroupOpType;
+import org.apache.thrift.TDeserializer;
+import org.apache.thrift.TException;
+import org.apache.thrift.TSerializer;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+import java.util.zip.CRC32;
+
+// WorkGroupEntry is used by EditLog to persist WorkGroupOp in replicated log
+// its format as follows
+// |--CRC(long)--|--Length(int)--|--TWorkGroupOp(Byte[])--|
+//
+public class WorkGroupEntry implements Writable {
```

Comment:
For new FE metadata, should Serde by json format.

Reply:
new fields maybe added to WorkGroup, thrift can handle it, json cannot .

Reply:
ok